docs/source/models/cause.rst

```diff
@@ -558,6 +560,97 @@ below.
+  appropriate for diarrhea, as there is a uniform remission rate across all 
+  cases. With diabetes, however, the remission rate does *not* tell us the 
+  average duration that an individual will experience diabetes.
+
+  How do we apply this to our simulants? Say we randomly selected 10 people with 
+  diarrhea in the Philippines on a random day in 2017. In the next day, they 
+  would accumulate 10 person days. Our rate tells us that in the next day, the 
+  expected value for cases remitted is given by:
+
+  .. math::
+
+    \frac{\text{1 case of diarrhea}}{\text{5 person-days}}\times\text{10 person-days} = \text{2 remitted cases of diarrhea}.
+
+  Similarly, we can take the rate of remission of diabetes, and for a randomly 
+  selected case of diabetes in Moldova in 2017, consider if they will remit some 
+  time in the next year. The expected value for cases remitted is then given by: 
+
+.. math::
+
+  \frac{\text{0.0036 cases of diabetes}}{\text{1 person-year}}\times\text{1 person-year} = \text{0.0036 remitted cases of diabetes}.
+
+Note that when we refer to remission rates, we are typically considering
+a rate within the infected or with-condition population. This is true both in
+general, and in the context of GBD--unlike with incidence, which GBD calculates 
+within the entire population, as discussed above.
+
+Remission within GBD
+""""""""""""""""""""
+Most nonfatal models in GBD are run using DisMod (:ref:`2017_cause_models`). 
+DisMod estimates compartmental models of disease, and thus produces estimates of 
+measures (prevalence, incidence, remission, excess mortality rate, etc.) that 
+are internally consistent for any given model. DisMod estimates remission rates 
+as:
+
+.. math::
+
+  \frac{\text{number of remitted cases}}{\text{person-years in the with-condition population}}
+
+.. todo::
+  update link to dismod page, once available
+
+GBD's final outputs, however, are in the form of YLLs, YLDs, and DALYs. To 
+calculate these measures such that they are consistent across *different* 
+causes, GBD runs standardizing processes on estimates of prevalence, incidence, 
+and estimated mortality rate. Note then that these final estimates are no longer 
+consistent with the DisMod estimates. However, as remission is not needed to 
+calculate YLDs, the latest-stage estimate of remission produced by GBD comes 
+from DisMod models.
+
+Implementing remission rates in cause models
+""""""""""""""""""""""""""""""""""""""""""""
+For a given simulation with timesteps of length *time unit* and a given time 
+unit, we convert remission rates to the form of cases remitted per 
+person-time-unit. If the rate is small with respect to the timestep (that is, if
+the rate is less than 1 per the time step), it can be used to compute the 
+probability of a simulant transitioning from an infected or with-condition state 
+to a susceptible or free-of-condition state in a given timestep.
```

Comment:
@NathanielBlairStahn do you think this is sufficiently clear?

Reply:
@beatrixkh Yeah, looks good!